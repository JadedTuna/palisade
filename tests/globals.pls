in {
    high a: int;
    low b: bool;
    high ah[5]: bool;
    low al[10]: int;
}
out {
    high x: bool;
    low y: int;
    high rh[15]: int;
    low rl[23]: bool;
}

// ensure globals are parsed properly
