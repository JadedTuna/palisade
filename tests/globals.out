[[1;32m OK [0m] [1;93mhigh[0m [1;34mx[0m is [1;93mhigh[0m
[[1;32m OK [0m] [1;93mlow [0m [1;34my[0m is [1;93mlow[0m
[[1;32m OK [0m] [1;93mhigh[0m [1;34mrh[0m is [1;93mhigh[0m
[[1;32m OK [0m] [1;93mlow [0m [1;34mrl[0m is [1;93mlow[0m
